examples/speaker_recognition/speaker_reco.py

```diff
@@ -43,15 +44,20 @@
 
 """
 
+seed_everything(42)
```

Comment:
@okuchaiev Do we want to encourage seeding? I believe some kernels of cuDNN anyway will be non-deterministic if you use cudnn autotune kernels.

Reply:
I suppose autotune is turned off by default so no harm done